app/components/base/buttons/clickable/clickable.component.ts

```diff
@@ -1,18 +1,48 @@
-import { Component, EventEmitter, HostBinding, HostListener, Input, Output } from "@angular/core";
+import {
+    Component, EventEmitter, HostBinding, HostListener,
+    Input, OnDestroy, Output, SimpleChanges,
+} from "@angular/core";
 
+import { AuthorizationHttpService, Permission } from "app/services";
+import { Subscription } from "rxjs";
 import "./clickable.scss";
 
 @Component({
     selector: "bl-clickable",
     template: `<ng-content></ng-content>`,
 })
-export class ClickableComponent {
+export class ClickableComponent implements OnDestroy {
     @Input() @HostBinding("class.disabled") public disabled = false;
 
     @Output() public do = new EventEmitter<Event>();
 
     @HostBinding("tabindex") public tabindex = "0";
     @HostBinding("class.focus-outline") public focusOutline = true;
+    public subtitle = "";
+    private _sub: Subscription;
+    public constructor(private authHttpService: AuthorizationHttpService) {
+    }
+
+    public onPermissionChange(changes: SimpleChanges): void {
+        this.subtitle = "";
+        if (changes.permission) {
+            this._clearSubscription();
+            if (changes.permission.currentValue === "write") {
+                this._sub = this.authHttpService.getResourcePermission().subscribe((permission: Permission) => {
+                    // when user only got 'read' permission
+                    if (permission === "read") {
```

Comment:
if(this.permission === "write" && permission === "read") {
// disable
} else {
// enable
}